File: jdroid-java-core/src/main/java/com/jdroid/java/utils/RandomUtils.java
package com.jdroid.java.utils;

import java.util.Random;

public class RandomUtils {

	private static final Random RANDOM = new Random();

	public static Long getLong() {
		return Math.abs(RANDOM.nextLong());
	}

	public static Integer getInt() {
		return Math.abs(RANDOM.nextInt());
	}

	public static Integer get16BitsInt() {
		return Math.abs(RANDOM.nextInt(16));
	}

	public static Integer getInt(int bound) {
		return Math.abs(RANDOM.nextInt(bound));
	}
}

File: jdroid-java-core/src/main/java/com/dipien/java/utils/RandomUtils.kt
package com.dipien.java.utils

import java.util.Random
import kotlin.math.abs

object RandomUtils {

    private val RANDOM = Random()

    fun getLong(): Long {
        return abs(RANDOM.nextLong())
    }

    fun getInt(): Int {
        return abs(RANDOM.nextInt())
    }

    fun get16BitsInt(): Int {
        return abs(RANDOM.nextInt(16))
    }

    fun getInt(bound: Int): Int {
        return abs(RANDOM.nextInt(bound))
    }
}
